controller/README.md

```diff
@@ -84,3 +84,247 @@ python benchmark.py --config example-config.yaml
 ```
 
 A separate *tmux* session (`benchmark-<name>`) is created for every model so you can watch latency/throughput side by side.
+
+## Testing Scripts
```

Comment:
The README here is to describe the functionality of the controller. You should describe the sleep model and traffic monitoring here, could be with some examples to test it, but not the entire test suite. All the tests should be moved to the test folder under project root.